riverlog/common_test.go

```diff
@@ -0,0 +1,41 @@
+package riverlog_test
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/riverqueue/river"
+	"github.com/riverqueue/river/rivershared/riversharedtest"
+	"github.com/riverqueue/river/rivershared/util/sliceutil"
+	"github.com/riverqueue/river/rivertype"
+)
+
+//
+// This file used as a holding place for test helpers for examples so that the
+// helpers aren't included in Godoc and keep each example more succinct.
+//
+
+// Wait on the given subscription channel for numJobs. Times out with a panic if
+// jobs take too long to be received.
+func waitForNJobs(subscribeChan <-chan *river.Event, numJobs int) []*rivertype.JobRow {
```

Comment:
not worth putting this into an internal pkg to share it vs copy/pasting?

Reply:
Yeah, I'll put it in a `rivershared` thing. We needed it in a pro example test before too.